### PR TITLE
Introduced github-templates

### PR DESCRIPTION
**Pull request by ansrivas**
Once merged, this is how it would look like:

### Proposed changes in this pull request
Introduced github templates for issue/pull-request

### Type (put an `x` where ever applicable)
- [ ] Bug fix: Link to the issue
- [ ] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement
- [x] Other

### Checklist
Please put an `x` against the checkboxes. Write a small comment explaining if its `N/A` (not applicable)

- [x] All the tests are passing after the introduction of new changes.
- [ ] Added tests respective to the part of code I have written.
- [x] Added proper documentation where ever applicable (in code and README.md).

### Optional extra information
N/A

### CODE REVIEW

**Comment by lornajane** [comment]
Could we rebase this to update with the new master changes please?

**Comment by ansrivas** [comment]
> Could we rebase this to update with the new master changes please?

Hi @lornajane, done.

**Comment by lornajane** [comment]
Would it be worth using different templates for the feature/bug/docs/other distinction? GitHub recommend doing it this way (https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) should we consider it here?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+### Description of the issue
+Describe the issue you are experiencing.
+
+### Issue-Type (put a `x` sign in the square brackets)
```
Comment (jankatins) [inline .github/ISSUE_TEMPLATE.md:4]: `... and delete the rest` ?

**Comment by ansrivas** [comment]
> Would it be worth using different templates for the feature/bug/docs/other distinction? GitHub recommend doing it this way (https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) should we consider it here?

Sure, I created this as a very basic template to ensure that users have a defined way to proceed. ( In case this fails to work, we can simply change it )

**Review by lornajane — APPROVED** [review]
Good starting point

**Pull request merged**